requirements.txt

```diff
@@ -1,18 +1,19 @@
 appdirs==1.4.4
-arrow==0.15.5
+arrow==0.17.0
 astroid==2.4.2
-attrs==20.2.0
-bandit==1.6.2
+attrs==20.3.0
+bandit==1.7.0
 black==20.8b1
-certifi==2020.6.20
-chardet==3.0.4
+certifi==2020.12.5
+chardet==4.0.0
 click==7.1.2
-codecov==2.1.10
+codecov==2.1.11
 colorama==0.4.4
-coverage==5.3
+coverage==5.3.1
 gitdb==4.0.5
-gitlint==0.14.0
-GitPython==3.1.11
+gitdb2==4.0.2
+gitlint==0.15.0
+GitPython==3.0.6
```

Comment:
Was the downgrade from 3.1.11 to 3.0.6 intentional? Due to a vuln?

Reply:
No it's just how they were generated, likely as a result the wnew pip dependency resolution changes that recently went live.  The dependencies of our project only specify `GitPython>=1.0.1`